ArmPkg/Library/ArmSmcLib/AArch64/ArmSmc.masm

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Does this need a header block & License?

Reply:
Yes, I'll add one.

Reply:
Done.